ui/src/core/xibo-forms.js

```diff
@@ -532,35 +718,177 @@ function membersFormSubmit(id) {
         return;
     }
 
+    // Create a new queue.
+    window.queue = $.jqmq({
```

Comment:
Can you explain why we have a queue here?

Reply:
Is it because the displays and display groups are shown in separate tabs on that form, and the save is taking care of both (submitting one after the other).

If that is still the case then I am happy to leave it!